azurerm/provider.go

```diff
@@ -116,6 +116,7 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_builtin_role_definition":                dataSourceArmBuiltInRoleDefinition(),
 			"azurerm_cdn_profile":                            dataSourceArmCdnProfile(),
 			"azurerm_client_config":                          dataSourceArmClientConfig(),
+			"azurerm_kubernetes_service_versions":            dataSourceArmKubernetesServiceVersions(),
```

Comment:
_minor_ can we sort this alphabetically?

Reply:
Sorry about that. I forgot it in the same position since the orchestrator version.
BTW, initially I did use the editor sort feature to sort the entire map and a few others are out of place. Didn’t want to pollute the PR, but worth checking.